runtime/src/main/java/org/corfudb/runtime/collections/CorfuStore.java

```diff
@@ -0,0 +1,130 @@
+package org.corfudb.runtime.collections;
+
+import com.google.protobuf.Message;
+
+import java.lang.reflect.InvocationTargetException;
+import java.util.Collection;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import org.corfudb.protocols.wireprotocol.Token;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.CorfuStoreMetadata.TableName;
+import org.corfudb.runtime.CorfuStoreMetadata.Timestamp;
+
+/**
+ * Created by zlokhandwala on 2019-08-02.
+ */
+public class CorfuStore {
```

Comment:
Why we even have this class? It's just a wrapper for TableRegistry, it doesn't contain any valuable logic. 
I don't see any reason to have this class in the code